source/clear-linux/tutorials/nvidia.rst

```diff
@@ -1,17 +1,18 @@
 .. _nvidia:
 
-Install NVIDIA Drivers
+Install NVIDIA\* Drivers
 ######################
```

Comment:
@DougTW. Overall looks great.  Please add more `#` below doc title. It needs this for a successful build.